src/main/java/com/cleanroommc/groovyscript/compat/mods/tinkersconstruct/Alloying.java

```diff
@@ -13,8 +17,10 @@
 import java.util.Arrays;
 import java.util.List;
 
+@RegistryDescription
 public class Alloying extends VirtualizedRegistry<AlloyRecipe> {
 
+    @RecipeBuilderDescription(example = @Example(".fluidOutput(fluid('iron') * 3).fluidInputs(fluid('clay') * 1,fluid('lava') * 2).register()"))
```

Comment:
`fluidInputs` isnt a method, only `fluidInput`

```suggestion
    @RecipeBuilderDescription(example = @Example(".fluidOutput(fluid('iron') * 3).fluidInput(fluid('clay') * 1,fluid('lava') * 2).register()"))
```